editor/src/com/talosvfx/talos/editor/notifications/commands/CommandsSystem.java

```diff
@@ -41,8 +41,7 @@ private boolean checkCommandState() {
         boolean isRun = false;
         for (ICommand command : allCommands) {
             if (command.isReadyToRun()) {
-                runCommand(command);
-                isRun = true;
+                isRun = runCommand(command);
```

Comment:
Should be checking if any have run, this will flag as false if another command comes later that doesn't run

Reply:
fixed.